libraries/RW/Artifactory/__init__.py

```diff
@@ -0,0 +1 @@
+from .Artifactory import Artifactory
```

Comment:
Does this actually do anything?

Reply:
It allows robot files to import via:
RW.Artifactory
vs
RW.Artifactory.Artifactory

and we can name the class files sane names instead of __init__.py